src/App.css

```diff
@@ -0,0 +1,22 @@
+.app {
+  max-width: 1200px;
+  margin: 0 auto;
+}
+.app .primary-btn a {
```

Comment:
- [ ] Always make sure to use specific classnames for all elements. Avoid using an element by itself even if its called inside another class

Reply:
- [ ] I can still see the css applied to the element. Change the CSS to a class name instead of an `a` tag.